guide/installation-and-configuration/upgrading.markdown

```diff
@@ -111,6 +116,19 @@ versions) below.
 5. Use `cf-promises` to verify that the policy runs with you previous version of CFEngine (e.g. 3.7), by running the same commands as above on a node with that CFEngine version.
 6. The merged masterfiles should now be based on the {{site.cfengine.branch}} framework, include your policies and work on both the version you are upgrading from and with {{site.cfengine.branch}}.
 
+## Upgrade from 3.6
+
+As 3.6 policy is not compatible with {{site.cfengine.branch}} some additional steps must be performed 
+to fulfill the upgrade procedure.
+
+1. From the version 3.9 legacy package promise is deprecated and new package promise is used by default.
+   There are ```body common control``` ```package_inventory``` and ```package_module``` attributes which are
```

Comment:
single **`** should suffice.

Perhaps we should like to the legacy package promise implementation here.

Beginning with version 3.9 the [Masterfiles Policy Framework][The Policy Framework] defaults to the new packages promise implementation for inventory of [packages installed][`packagesmatching()`] and [packages updates][`packageupdatesmatching()`]. See [package_inventory] in `body common control` for details on modifying the default sources for package inventory.